telegram/chat.py

```diff
@@ -92,14 +92,14 @@ class Chat(TelegramObject):
 
     """
 
-    PRIVATE: str = 'private'
-    """:obj:`str`: 'private'"""
-    GROUP: str = 'group'
-    """:obj:`str`: 'group'"""
-    SUPERGROUP: str = 'supergroup'
-    """:obj:`str`: 'supergroup'"""
-    CHANNEL: str = 'channel'
-    """:obj:`str`: 'channel'"""
+    PRIVATE: str = constants.CHAT_PRIVATE
+    """:const:`telegram.constants.CHAT_PRIVATE`"""
```

Comment:
Did you build the docs to check that this renders correctly?

Reply:
Yes I built. Everything is displayed correctly and links work.

Reply:
👍